packages/view-util/query/cubeQuery.js

```diff
@@ -0,0 +1,7 @@
+import { sparql } from '@tpluscode/sparql-builder'
+import { VALUES } from '@tpluscode/sparql-builder/expressions'
+
+export function cubeQuery(query, ...values) {
+  return sparql`${query}
+  ${VALUES(...values)}`
+}
```

Comment:
Queries can be easily parametrised by adding `VALUES` **after** its body

This simply concatenates the loaded `query: string` with a `VALUES` generated from the `values` array of objects